### PR TITLE
minor update and fixed some errors

### PR DESCRIPTION
**Pull request by Emily-Jiang**

### CODE REVIEW

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 1]
```diff
@@ -35,35 +35,32 @@ The specifications for each release are listed in reverse chronological order.
...
- - https://github.com/eclipse/microprofile-fault-tolerance/releases/tag/4.0.1[MicroProfile Fault Tolerance 4.0]
+ - https://github.com/eclipse/microprofile-fault-tolerance/releases/tag/4.0.2[MicroProfile Fault Tolerance 4.0]
  - https://github.com/eclipse/microprofile-metrics/releases/tag/5.0[MicroProfile Metrics 5.0]
  - https://github.com/eclipse/microprofile-health/releases/tag/4.0.1[MicroProfile Health 4.0]
  - https://github.com/eclipse/microprofile-telemetry/releases/tag/1.0[MicroProfile Telemetry 1.0]
  - https://github.com/eclipse/microprofile-open-api/releases/tag/3.1[MicroProfile OpenAPI 3.1]
- - https://github.com/eclipse/microprofile-jwt-auth/releases/tag/2.0[MicroProfile JWT Authentication 2.0] (TODO: Update to 2.1?)
+ - https://github.com/eclipse/microprofile-jwt-auth/releases/tag/2.1[MicroProfile JWT Authentication 2.1]
  - https://github.com/eclipse/microprofile-rest-client/releases/tag/3.0.1[MicroProfile Rest Client 3.0]
- - https://jakarta.ee/specifications/coreprofile/10/[Jakarta JSON Processing 2.0]
```
[the remark this answers is not in the document]
Comment (Emily-Jiang) [reply spec/src/main/asciidoc/architecture.asciidoc:66]: @njr-11 please see the line 49 as I moved it to the top.

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 2]
```diff
@@ -35,35 +35,32 @@ The specifications for each release are listed in reverse chronological order.
...
-* https://download.eclipse.org/microprofile/microprofile-fault-tolerance-4.0.1/microprofile-fault-tolerance-spec-4.0.1.html#_incompatible_changes[Fault Tolerance]
-* https://download.eclipse.org/microprofile/microprofile-health-4.0.1/microprofile-health-spec-4.0.1.html#_incompatible_changes[Health]
-* https://download.eclipse.org/microprofile/microprofile-metrics-5.0/microprofile-metrics-spec-5.0.html#_incompatible_changes[Metrics]
-* https://download.eclipse.org/microprofile/microprofile-open-api-3.1/microprofile-openapi-spec-3.1.html#_incompatible_changes[OpenAPI]
-* https://download.eclipse.org/microprofile/microprofile-rest-client-3.0.1/microprofile-rest-client-spec-3.0.1.html#_incompatible_changes[Rest Client]
-* https://download.eclipse.org/microprofile/microprofile-jwt-auth-2.0/microprofile-jwt-auth-spec-2.0.html#_incompatible_changes[JWT] (TODO - Update to JWT 2.1?)
-* https://download.eclipse.org/microprofile/microprofile-telemetry-1.0/microprofile-telemetry-spec-1.0.html#_incompatible_changes[Telemetry]
-
+If you are still dependent on Jakarta EE 9.1, please consider using the https://github.com/eclipse/microprofile/releases/tag/5.0[5.0 release of MicroProfile].
```
Comment (ederks85) [inline spec/src/main/asciidoc/architecture.asciidoc:42]: This is an important addition, this must remain in future releases

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 3]
```diff
@@ -35,35 +35,32 @@ The specifications for each release are listed in reverse chronological order.
...
- - https://github.com/eclipse/microprofile-jwt-auth/releases/tag/2.0[MicroProfile JWT Authentication 2.0] (TODO: Update to 2.1?)
+ - https://github.com/eclipse/microprofile-jwt-auth/releases/tag/2.1[MicroProfile JWT Authentication 2.1]
  - https://github.com/eclipse/microprofile-rest-client/releases/tag/3.0.1[MicroProfile Rest Client 3.0]
- - https://jakarta.ee/specifications/coreprofile/10/[Jakarta JSON Processing 2.0]
+
+MicroProfile 6.0 has the following backward incompatible changes compared to MicroProfile 5.0.
+
+* https://download.eclipse.org/microprofile/microprofile-metrics-5.0.0/microprofile-metrics-spec-5.0.0.html#_incompatible_changes[MicroProfile Metrics Incompatible changes]
+* MicroProfile OpenTracing replaced by MicroProfile Telemetry
+* Based on Jakarta EE 10 Core Profile instead of the five specifications in Jakarta EE 9.1
```
Comment (ederks85) [inline spec/src/main/asciidoc/architecture.asciidoc:63]: Seen the discussion we're having on the mailing list, is "based" the correct wording here? It if clear to me, but it must also be to others. And let's use that terminology in future releases then

**Comment by ederks85** [comment]
Thanks @Emily-Jiang, if you see no futher changes necessary based on my comments, I'm fine with the changes

File: spec/src/main/asciidoc/architecture.asciidoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (njr-11) [reply spec/src/main/asciidoc/architecture.asciidoc:66]: Thanks! I missed that. I'll mark this resolved.

**Review by njr-11 — APPROVED** [review]

File: spec/src/main/asciidoc/architecture.asciidoc (hunk 3 above)
Comment (Emily-Jiang) [reply spec/src/main/asciidoc/architecture.asciidoc:63]: I think this is accurate word for me as well. We can update this if someone comes up a better one.

Comment (njr-11) [reply spec/src/main/asciidoc/architecture.asciidoc:63]: The term "Based on" is vague in its meaning.  More precise wording would be "Includes the ..." to be consistent with the section, "The complete list of MicroProfile 6.0 specifications includes..."
But even we leave it as the more vague "based on", the presence of that of later statement should clarify the intent.

**Pull request merged**